<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>611</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1001</width>
      <height>521</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::NoAnchor</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRigth">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Полотно : 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество : 1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>180</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Кисть</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прерывистый прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прерывистая линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Окружность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прперывистая окружность</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ластик</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>30</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionImage"/>
     <addaction name="actionImage_list"/>
     <addaction name="actionPDF_2"/>
    </widget>
    <addaction name="actionCreate_new_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
    <addaction name="actionClear_canvas"/>
    <addaction name="actionSet_background_color"/>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Лицензия</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Создать новый проект</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Набор изображений</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionCreate_new_project">
   <property name="text">
    <string>Create new project</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionImage_list">
   <property name="text">
    <string>Image list</string>
   </property>
  </action>
  <action name="actionPDF_2">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionclear_all">
   <property name="text">
    <string>clear list</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="actionClear_canvas">
   <property name="text">
    <string>Очистить фон</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сглаживание</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Верифицировать</string>
   </property>
  </action>
  <action name="actionSet_background_color">
   <property name="text">
    <string>Цвет фона</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Сменить кисть</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Сменить кисть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
